sample/app/http-env-echo-no-loadbalancer.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Does not exist yet; automated executions of the suite would be ~8h to setup (with the latter ~6h just waiting for the tests to run 2-3 times to cleanup bugs once they're done).

The idea would be to define a second `cloud` in Jenkins-EC2 terminology (https://wiki.jenkins-ci.org/display/JENKINS/Amazon+EC2+Plugin).
That cloud would launch slaves in a non-privileged account, with unique `label`s compared to our current slaves. These slaves would _only_ run the integration tests (or eventually: deployments in that non-privileged account).